### PR TITLE
Participant Andreas Sahle

### PR DESCRIPTION
**Pull request by pixelmord**
Not easy to create a PR with an IPad on holidays ;)
Involved a lot of copy and paste wizardry

I would like to register for JS CraftCamp.    
- [x] My Pull Request contains a json file `$firstname_$lastname.json` or `$nickname.json`    
- [x] The json file follows the template at https://github.com/jscraftcamp/website/blob/master/participants/_template.json and contains the mandatory fields `name`, `tags`, `when`, `what_is_my_connection_to_javascript` and `what_can_i_contribute`    
- [x] If I can't attend, I will either send another Pull Request removing my json file or an E-Mail with the subject 'UNREGISTER' to team@jscraftcamp.org

### CODE REVIEW

File: participants/andreas_sahle.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
+          "name": "Andreas Sahle",
+          "company": "Wunder Germany",
+          "email": "andreas.sahle@wunder.io"
```
Comment (davelosert) [inline participants/andreas_sahle.json]: You are missing a comma here 😄

**Comment by davelosert** [comment]
@pixelmord : Thanks for your registration. That is maybe the first registration via IPad 😄 
Unfortunately there is one minor thing:  a missing comma in your registration json - could you please take the time to fix that?

Also would be great if you could check the checkboxes in your comment. Thanks a lot in advance.

**Comment by pixelmord** [comment]
Thanks for the review @davelosert 
I found the comma key on the touch keyboard ;) 
apple should really give us arrow keys, I really like that on android devices...

**Comment by davelosert** [comment]
@pixelmord : Thank you very much for taking the time to fix the issue.

Last year we did a little Ice-Breaker game in the beginning of JSCC where our participants were invited discuss controversial topics - but only by taking the OTHER side of their actual oppinion.
Anrdoid vs. IOS could be such a topic. 😄 

I am looking forward on meeting you at JSCC17!

**Pull request merged**